components/campaigns/banners.ts

```diff
@@ -15,18 +15,18 @@ function shouldShowBanner(cfpDeadline: string) {
   return true;
 }
 
-const cfpDeadlineParis = '2024-10-12T06:00:00Z';
+const cfpDeadlineParis = '2024-10-30T06:00:00Z';
```

Comment:
Instead of using this variable, can you please add this inside map? It is very confusing to understand why are we giving `cfpDeadlineParis` deadline to every banner 😅

Reply:
I'm lost @akshatnema please explain. Oooh I used default cfpparis because it was a quick change after the Paris banner 🫣🫣🫣